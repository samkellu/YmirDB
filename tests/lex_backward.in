set rat 123
set bat rat
set dog rat
set c bat
backward rat
backward bat
backward dog
bye
