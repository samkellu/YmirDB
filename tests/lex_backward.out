> ok

> ok

> ok

> ok

> bat, c, dog

> c

> nil

> bye
